specification/desktopvirtualization/DesktopVirtualization.Management/models.tsp

```diff
@@ -2523,6 +2546,18 @@ model HostPoolProperties {
    * The type of management for this hostpool, Automated or Standard. The default value is Automated.
    */
   managementType?: ManagementType;
+
+  /**
+   * DeploymentScope type for HostPool.
+   */
+  @added(Versions.v2025_08_01_preview)
+  deploymentScope?: DeploymentScope;
```

Comment:
This should have @visibility(Lifecycle.Read) to indicate that it is readonly.

Reply:
Added